### ISSUE TITLE
 [Vue warn]: Unknown custom element: <tabbar> - did you mean <tabbar>? HTML is case-insensitive, remember to use kebab-case in templates

### ISSUE THREAD

**Issue by ohohco** [airyland/vux#168]
I create the project by `vue-cli`, use the eslint by standard. The bottom code will error in console: ` [Vue warn]: Unknown custom element: <tabbar> - did you mean <tabbar>? HTML is case-insensitive, remember to use kebab-case in templates`
```
<template>
  <div>
    <tabbar>
      <tabbar-item selected>
        <!-- <img slot="icon" src="../assets/demo/icon_nav_button.png"> -->
        <span slot="label">Wechat</span>
      </tabbar-item>
      <tabbar-item show-dot>
        <!-- <img slot="icon" src="../assets/demo/icon_nav_msg.png"> -->
        <span slot="label">Message</span>
      </tabbar-item>
      <tabbar-item>
        <!-- <img slot="icon" src="../assets/demo/icon_nav_article.png"> -->
        <span slot="label">Explore</span>
      </tabbar-item>
      <tabbar-item>
        <!-- <img slot="icon" src="../assets/demo/icon_nav_cell.png"> -->
        <span slot="label">News</span>
      </tabbar-item>
    </tabbar>
  </div>
</template>

<script>
import Tabbar from 'vux/components/tabbar'
import TabbarItem from 'vux/components/tabbar-item'
export default {
  components: {
    Tabbar,
    TabbarItem
  }
}
</script>

```

**Comment by airyland** [comment]
Please specified vux npm version.

**Comment by ohohco** [comment]
npm info vux version
0.0.113

**Comment by airyland** [comment]
Thanks for your feedback. Already Fixed in v0.0.114.

**Comment by ohohco** [comment]
The same with the element `blur`, please have a look
